src/System/Net/Benchmarks/Apps/Socket/Client/Program.cs

```diff
@@ -0,0 +1,247 @@
+                    sw.Restart();
+                }
+
+                await stream.WriteAsync(sendBuffer, default).ConfigureAwait(false);
+                bytesWritten += bufferSize;
+            }
+
+            sw.Stop();
+            var elapsed = sw.ElapsedTicks * 1.0 / Stopwatch.Frequency;
+            return bytesWritten / elapsed;
+        }
+
+        async Task<double> ReadingTask(Stream stream, int bufferSize)
+        {
+            if (bufferSize == 0)
+            {
+                return 0;
+            }
+
+            var recvBuffer = new byte[bufferSize];
+            var sw = Stopwatch.StartNew();
+            var isWarmup = true;
+
+            long bytesRead = 0;
+
+            while (IsRunning)
+            {
+                if (isWarmup && !IsWarmup)
+                {
+                    if (bytesRead == 0)
+                    {
+                        throw new Exception($"No bytes read during warmup.");
+                    }
+                    isWarmup = false;
+                    bytesRead = 0;
+                    OnWarmupCompleted();
+                    sw.Restart();
+                }
+
+                bytesRead += await stream.ReadAsync(recvBuffer, default).ConfigureAwait(false);
+            }
+
+            sw.Stop();
+            var elapsed = sw.ElapsedTicks * 1.0 / Stopwatch.Frequency;
+            return bytesRead / elapsed;
+        }
+    }
+
+    private static async Task<(double Rps, long Errors)> RpsScenario(SocketClientConnection connection, SocketClientOptions options)
+    {
+        await using var stream = await connection.EstablishStreamAsync(options).ConfigureAwait(false);
+
+        var sendBuffer = new byte[options.SendBufferSize];
+        var receiveBuffer = new byte[options.ReceiveBufferSize];
+
+        long successRequests = 0;
+        long exceptionRequests = 0;
+
+        var isWarmup = true;
+        var sw = Stopwatch.StartNew();
+
+        while (IsRunning)
+        {
+            if (isWarmup && !IsWarmup)
+            {
+                if (successRequests == 0)
+                {
+                    throw new Exception($"No successful requests during warmup.");
+                }
+                isWarmup = false;
+                successRequests = 0;
+                exceptionRequests = 0;
+                OnWarmupCompleted();
+                sw.Restart();
+            }
+
+            try
+            {
+                await stream.WriteAsync(sendBuffer, default).ConfigureAwait(false);
+                await stream.ReadExactlyAsync(receiveBuffer, 0, receiveBuffer.Length, default).ConfigureAwait(false);
+                successRequests++;
+            }
+            catch (Exception ex)
+            {
+                Log("Exception: " + ex);
+                exceptionRequests++;
+            }
+        }
+        var elapsed = sw.ElapsedTicks * 1.0 / Stopwatch.Frequency;
+        return (successRequests / elapsed, exceptionRequests);
+    }
+}
```

Comment:
Can you pls unify the file endings so that all of the files end with a new line?